### DISCUSSION TITLE
installation without docker

### DISCUSSION THREAD

**Discussion by riteshmk** [magda-io/magda#3259] [Q&A] [answered]
Hi Gurus,

could you help me if , its possible to install and maintain Magda without docker ? i don't intend to use docker.
appreciate your support.

Cheers!!

**Comment by t83714** [comment, marked as answer]
Magda is a cloud-native app that requires k8s to run.
That's how we can make it a hardware/environment independent solution.
It's not hard to run a local k8s env --- for local dev, you probably can try [minikube](https://minikube.sigs.k8s.io/docs/start/)
For production, you probably don't need to install a k8s cluster. Choosing one of the cloud providers (AWS, Google, Azure) would be a common choice.
